src/placeos-driver/proxy/driver.cr

```diff
@@ -141,9 +141,9 @@ class PlaceOS::Driver::Proxy::Driver
                     index += 1
                     arguments[index - 1]
                   elsif details.size > 1
```

Comment:
This looks incorrect given move from Array.

Reply:
it's correct, the remote function calls are built via the `method_missing` macro so this is actually the tuple of arguments passed to the RPC - the `function` variable holds the interface metadata